src/main/webapp/app/shared/layouts/profiles/profile-info.model.ts

```diff
@@ -56,6 +56,8 @@ export class ProfileInfo {
         };
     };
     public theiaPortalURL: string;
+    public operatorName: string;
+    public operatorAdminName: string;
```

Comment:
The template of the `about-us.component` treats the operatorAdminName as optional. Is it optional?

```suggestion
    public operatorName: string;
    public operatorAdminName?: string;
```


----

Also, you'll have to update some test data in

`code-button.component.spec.ts` and `profile.service.spec.ts`, where `operatorName` and `operatorAdminName` are currently missing & causing the client test build to fail